rpc/modules/api.go

```diff
@@ -0,0 +1,21 @@
+package modules
+
+import (
+	"github.com/ChainSafe/gossamer/common"
+)
+
+// StorageAPI ...
```

Comment:
Can we not add (...) as a valid description for public exported structs ? would be nice to have a proper explanation of why this is useful and what it does. Same for the others you have added in this PR.

Reply:
done